src/frontends/pytorch/src/op/norm.cpp

```diff
@@ -259,11 +259,8 @@ OutputVector translate_linalg_norm(const NodeContext& context) {
             auto input_rank = x.get_partial_shape().rank();
             if (input_rank.is_static() && input_rank.get_length() == 2) {
                 result = frobenius_norm(context, x, dim, keep_dim);
-            } else if (input_rank.is_static() && input_rank.get_length() == 1) {
-                result = norm_vector(context, x, dim, 2, keep_dim);
             } else {
-                FRONT_END_OP_CONVERSION_CHECK(false,
```

Comment:
Does it change the behaivior or just allows dynamic rank? If the later I think it makes sense to reverse the validation. Previously it was working for `input_rank.is_static() && input_rank.get_length() == 1`. Replace it with working if `input_rank.is_dynamic() || input_rank.get_length() == 1`

Reply:
I'm not sure that it is my case, I'll check. I thought that in my case rank known but it is greater 2

Reply:
yes, it works in my case, updated